packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -285,6 +306,12 @@
     try {
       loading = true;
 
+      statuses.push({
+        name: "APPROVE_ERC20",
+        message: "Approve ERC20",
+        done: false,
+      });
+
```

Comment:
I'm concerned about adding a state boolean dynamically. I agree with CodeRabbit that this should be defined statically.

Reply:
Well the problem is that we will see it every time even if we don't have something to approve, so that is why it is added dynamically